### PR TITLE
[mqtt] Pump subscribes to initial topics for 

### PR DESCRIPTION
**Pull request by dmolokanov**
Implemented missed subscriptions to RPC command topic from Local Pump event handler. 
Refactor to **automatically** subscribe to default topics for both Pumps subscriptions from default settings and for RPC command

### CODE REVIEW

File: mqtt/mqtt-bridge/src/pump/ingress.rs [hunk 1]
```diff
@@ -57,5 +57,7 @@ impl IngressShutdownHandle {
 }
 
 #[derive(Debug, thiserror::Error)]
-#[error("ingress error")]
-pub(crate) struct IngressError;
+pub(crate) enum IngressError {
+    #[error("mqtt client error")]
```
Comment (vadim-kovalyov) [inline mqtt/mqtt-bridge/src/pump/ingress.rs:61]: `{0}`

File: mqtt/mqtt-bridge/src/client.rs [hunk 2]
```diff
@@ -313,15 +321,14 @@ where
         }
     }
 
-    pub async fn subscribe(&mut self, topics: &[String]) -> Result<(), ClientError> {
-        info!("subscribing to topics");
-        let subscriptions = topics.iter().map(|topic| proto::SubscribeTo {
-            topic_filter: topic.to_string(),
+    async fn subscribe(&mut self, topics: Vec<String>) -> Result<(), ClientError> {
```
Comment (ancaantochi) [inline mqtt/mqtt-bridge/src/client.rs:324]: Why did you change to Vec? I was using Vec initially and I think clippy complained about using &[String]

File: mqtt/mqtt-bridge/src/client.rs [hunk 3]
```diff
@@ -313,15 +321,14 @@ where
...
+        debug!("subscribing to topics {:?}...", topics);
```
Comment (ancaantochi) [inline mqtt/mqtt-bridge/src/client.rs:325]: I think it should be logged as info

File: mqtt/mqtt-bridge/src/messages.rs [hunk 4]
```diff
@@ -84,15 +84,13 @@ impl<S> StoreMqttEventHandler<S> {
...
+            warn!("unexpected subscription ack for {}", sub);
         };
     }
 
     fn update_unsubscribed(&mut self, sub: &str) {
-        if self.topic_mappers_updates.contains_key(sub) {
-            self.topic_mappers.remove(sub);
-        } else {
-            debug!("unsubscription ack for {} not expected", sub);
+        if self.topic_mappers_updates.remove(sub).is_none() {
```
Comment (ancaantochi) [inline mqtt/mqtt-bridge/src/messages.rs:92]: it should be removed from topic_mappers, not from topic_mappers_updates

Comment (ancaantochi) [reply mqtt/mqtt-bridge/src/messages.rs:92]: that shows that there is no test for unsubscribe, can you please add or I ca add one

File: mqtt/mqtt-bridge/src/client.rs (hunk 2 above)
Comment (dmolokanov) [reply mqtt/mqtt-bridge/src/client.rs:324]: I refactored that to consume vec without cloning elements. That's why clippy is happy now :)

File: mqtt/mqtt-bridge/src/messages.rs (hunk 4 above)
Comment (dmolokanov) [reply mqtt/mqtt-bridge/src/messages.rs:92]: Yeah, this is a mistake here

**Review by ancaantochi — APPROVED** [review]

File: mqtt/mqtt-bridge/src/client.rs [hunk 5]
```diff
@@ -8,7 +8,7 @@ use futures_util::future::{self, BoxFuture};
 use mockall::automock;
 use openssl::{ssl::SslConnector, ssl::SslMethod, x509::X509};
 use tokio::{io::AsyncRead, io::AsyncWrite, net::TcpStream, stream::StreamExt};
-use tracing::{debug, error, info};
```
Comment (and-rewsmith) [inline mqtt/mqtt-bridge/src/client.rs:11]: This removal is failing the build

**Review by vadim-kovalyov — APPROVED** [review]

File: mqtt/mqtt-bridge/src/client.rs (hunk 5 above)
Comment (dmolokanov) [reply mqtt/mqtt-bridge/src/client.rs]: yep...

**Review by vadim-kovalyov — APPROVED** [review]
